provider/runbook_data.go

```diff
@@ -35,6 +37,10 @@ func dataSourceRunbook() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+			"attachment_rule": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
```

Comment:
nit: could this be moved up so that it is at the top of the computed attributes? i'm trying to keep everything following the pattern of required then optional then computed, all in alphabetical order